src/Panes/TerminalPane.vala

```diff
@@ -44,6 +45,7 @@ namespace ElementaryTweaks {
             natural_copy_paste = box.add_switch (_("Natural copy paste"));
             follow_last_tab = box.add_switch (_("Follow last tab"));
             unsafe_paste_alert = box.add_switch (_("Unsafe paste alert"));
+            rem_tabs = box.add_switch (_("Remember tabs"));
```

Comment:
Here is where you need to make the swap for the paste alert. Just insert this line on line 47 instead :)